src/pages/settings/Profile/CustomStatus/StatusClearAfterPage.js

```diff
@@ -166,6 +166,7 @@ function StatusClearAfterPage({currentUserPersonalDetails, customStatus}) {
                     key={`${index}+${item.value}`}
                     onSelectRow={() => updateMode(item)}
                     showTooltip={false}
+                    isFocused={item.isSelected}
```

Comment:
Shouldn't we use the `SelectionList` instead? As we do in the `PriorityModePage`?

Reply:
`SelectionList` under the hood uses the same component as I did. `SelectionList` is a wrap that provides search input and other functionalities. So, the theme switching or style changes will also apply to it.